components/Recipe/recipeTeaser-component.tsx

```diff
@@ -0,0 +1,44 @@
+import stylesHp from '../../styles/Homepage/Homepage.module.css'
+import Recipe,{RecipeResults} from "../../types/Recipe/recipe-type";
+import Link from 'next/link'
+
+type Props = {
+    allRecipes: RecipeResults;
+}
+const RecipeTeaserComponent = ({allRecipes}:Props) => {
+    const recipes = allRecipes.results;
+    
+    return(
+        <div>
+            <style jsx>{`
```

Comment:
I would prefer to only one styling paradigm in the app instead of some inline and some under the /styles folder. With how much is already in the styles/ folder this could probably be moved there. Same with components/Recipe/recipe-component.tsx